File: Frontend/library-app/src/models/PendingPaymentRequest.ts
class PendingPaymentRequest {
  bookId: number;
  amount: number;
  cartItemId: number;

  constructor(book_id: number, amount: number, cartItemId: number) {
    this.bookId = book_id;
    this.amount = amount;
    this.cartItemId = cartItemId;
  }
}
export default PendingPaymentRequest;

File: src/main/java/com/reactlibraryproject/springbootlibrary/RequestModels/PendingPaymentRequest.java
package com.reactlibraryproject.springbootlibrary.RequestModels;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class PendingPaymentRequest {

  private Long bookId;
  private int amount;
  private Long cartItemId;
}
